packages/node_modules/@node-red/nodes/core/storage/10-file.html

```diff
@@ -342,7 +343,8 @@
             }).text(label).appendTo(encSel);
             $("#node-input-filename").typedInput({
                 default: "msg",
-                types:[{ value: "str", label:"", icon:"red/images/typedInput/az.svg"}, "msg", "jsonata", "env"],
+                types:[{ value: "str", label: RED._("common.type.string"),
+                         icon:"red/images/typedInput/az.svg"}, "msg", "jsonata", "env"],
```

Comment:
```suggestion
                types:["str", "msg", "jsonata", "env"],
```

Same comment as for the `file` node..